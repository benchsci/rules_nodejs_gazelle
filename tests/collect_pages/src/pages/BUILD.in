# gazelle:js_nextjs_pages_root
