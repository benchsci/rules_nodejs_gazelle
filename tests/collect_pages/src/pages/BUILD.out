load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

# gazelle:js_nextjs_pages_root

ts_project(
    name = "pages",
    srcs = [
        "404.ts",
        "index.ts",
    ],
    deps = ["//src/pages/my_page"],
)
